src/ready/apis/plot_losses.py

```diff
@@ -19,42 +19,44 @@
+
+        directory, csv_file = os.path.split(current_training_loss_file)
+        current_training_loss_df = pd.read_csv(path_current_training_loss, names=[str(current_training_loss_file)])
 
-    path_filename1 = os.path.join(MODELS_PATH, config.losses.loss_file1)
-    path_filename2 = os.path.join(MODELS_PATH, config.losses.loss_file2)
-    path_filename3 = os.path.join(MODELS_PATH, config.losses.loss_file3)
-    path_filename4 = os.path.join(MODELS_PATH, config.losses.loss_file4)
-    path_filename5 = os.path.join(MODELS_PATH, config.losses.loss_file5)
+        current_training_loss_df['epochs'] = current_training_loss_df.index
+        
+        training_loss_label = "training_loss_" + str(directory) 
+        plt.plot(current_training_loss_df['epochs'], current_training_loss_df[str(current_training_loss_file)],label=training_loss_label, linewidth=3) 
 
-    df1 = pd.read_csv(path_filename1, names=['lf1'], header=None)
-    df2 = pd.read_csv(path_filename2, names=['lf2'], header=None)
-    df3 = pd.read_csv(path_filename3, names=['lf3'], header=None)
-    df4 = pd.read_csv(path_filename4, names=['lf4'], header=None)
-    df5 = pd.read_csv(path_filename5, names=['lf5'], header=None)
+    for current_validation_loss_file in VALIDATION_LOSS_FILES:
 
-    df1['epochs'] = df1.index
-    df1['lf2'] = df2
-    df1['lf3'] = df3
-    df1['lf4'] = df4
-    df1['lf5'] = df5
+        path_current_validation_loss = os.path.join(MODELS_PATH, current_validation_loss_file)
 
-    logger.info(f"\n Dataframe: {df1}")
+        directory, csv_file = os.path.split(current_validation_loss_file)
+        current_validation_loss_df = pd.read_csv(path_current_validation_loss, names=[str(current_validation_loss_file)])
 
-    # plt.plot(df1['epochs'], df1['lf1'], df1['epochs'], df1['lf2'], df1['epochs'], df1['lf3'], df1['epochs'], df1['lf4'], df1['epochs'], df1['lf5'], linewidth=3)
-    plt.plot(df1['epochs'], df1['lf1'], df1['epochs'], df1['lf2'], df1['epochs'], df1['lf3'], linewidth=3)
-    # plt.title("Losses for models trained 100epochs in a100-80gb gpu")
+        current_validation_loss_df['epochs'] = current_validation_loss_df.index
 
+        validation_loss_label = "validation_loss_" + str(directory)
+        plt.plot(current_validation_loss_df['epochs'], current_validation_loss_df[str(current_validation_loss_file)], label=validation_loss_label, linewidth=3, linestyle='dashed')
+    
+    plt.title("Training and Validation Loss for each Epoch", fontsize=18)
     plt.xlabel("Epochs", fontsize=18)
     plt.ylabel("Loss", fontsize=18)
-    plt.tick_params(axis='both', labelsize=13)
-    # plt.legend(["naug_d1144", "waug_d1144", "waug_d0572", "waug_d0286", "waug_d0145"])
-    plt.legend(["naug_d1144", "waug_d1144", "waug_d0572"], fontsize=18)
-    plt.tight_layout()
+    plt.tick_params(axis='both', labelsize=13) 
+    plt.legend(framealpha=0.5)
     plt.grid()
+    # plt.tight_layout()
```

Comment:
DITTO Any difference with or without plt.tight_layout()? Maybe removed comment if there is no such difference